src/HttpRequest.cc

```diff
@@ -544,6 +544,8 @@ HttpRequest::maybeCacheable()
     if (!flags.hostVerified && (flags.intercepted || flags.interceptTproxy))
         return false;
 
+    /// XXX: handle squid-internal-mgr requests
+
```

Comment:
I an XXX here instead of adjusting/fixing the code: I guess that it should be resolved in PR1185.